data/helpers.d/utils

```diff
@@ -59,30 +59,35 @@ ynh_restore_upgradebackup () {
 # ynh_abort_if_errors
 #
 ynh_backup_before_upgrade () {
-	backup_number=1
-	old_backup_number=2
-	app_bck=${app//_/-}	# Replace all '_' by '-'
-	
-    	# Check if a backup already exists with the prefix 1
-	if sudo yunohost backup list | grep -q $app_bck-pre-upgrade1
-    	then
-        	# Prefix becomes 2 to preserve the previous backup
-		backup_number=2
-		old_backup_number=1
-	fi
+	if [ -e "/etc/yunohost/apps/$app/scripts/backup" ]
```

Comment:
That's a minor thing, but could we maybe simplify this to save 1 indent level ? e.g. :

```
if [ file does not exists ] 
then
     echo "This app doesn't have any backup script." >&2
     return
fi

...the rest of the function...
```